# HG changeset patch
# Parent 0d60587c3068d25c107ee81893791469b2623f0c
Make GET and CONNECT an alias for QUIT during the unregistered part of the server (like POST)

diff -r 0d60587c3068 ircd/parse.c
--- a/ircd/parse.c	Sun Jun 30 17:45:29 2013 +0200
+++ b/ircd/parse.c	Sun Jun 30 23:54:33 2013 +0200
@@ -419,7 +419,7 @@
     TOK_CONNECT,
     0, MAXPARA, MFLG_SLOW, 0, NULL,
     /* UNREG, CLIENT, SERVER, OPER, SERVICE */
-    { m_unregistered, m_not_oper, ms_connect, mo_connect, m_ignore }
+    { m_quit, m_not_oper, ms_connect, mo_connect, m_ignore }
   },
   {
     MSG_MAP,
@@ -594,7 +594,7 @@
     TOK_GET,
     0, MAXPARA, MFLG_SLOW, 0, NULL,
     /* UNREG, CLIENT, SERVER, OPER, SERVICE */
-    { m_unregistered, m_not_oper, m_ignore, mo_get, m_ignore }
+    { m_quit, m_not_oper, m_ignore, mo_get, m_ignore }
   },
   {
     MSG_PRIVS,
